### ISSUE TITLE
Only one of two subnets is advertised

### ISSUE THREAD

**Issue by andorardo** [hassio-addons/addon-tailscale#96]
# Problem/Motivation

I have two subnets on my HA host, on two NICs. `ha net info` shows as much.

They are 192.168.X.0/24 and 192.168.Y.0/24

## Expected behavior

In tailscale, that both subnets appear under "Subnet routes"

## Actual behavior

only the first subnet shows up under Subnet routes

## Steps to reproduce

Use two subnets on two NICs?

## Proposed changes

Either a way to detect both subnets, or a way to configure the `advertise-routes` option

**Comment by andorardo** [comment]
Please don't close this as "completed", since it is still very much an issue!

**Comment by dalkain** [comment]
Also interested in this being implemented

**Comment by SoulEater45** [comment]
It would be nice to have the possibility to enter further subnets in the config as an array. If I have time in the upcoming days, maybe I can try to configure it, since it should be just an array joined with a comma and slapped onto the `--advertise-routers` flag.

**Comment by DirschedlF** [comment]
I'm still interested to specify more than one subnet. See my comment regarding "--advertise-routes= ..." above.

**Comment by ocsi01** [comment]
I am also interested in this to be solved. Having both LAN and WIFI on my Hass devices, it would be nice to have both as shareable subnets.

**Comment by Mincka** [comment]
In the past, I used this alternative addon for tailscale, when this one was broken.
https://github.com/tsujamin/hass-addons/tree/main/tailscale

Since I need multiple route support now and there's no support for it, I'm switching back to @tsujamin's addon.
It offers a more recent version and a lot more possibilities regarding the options, especially route advertising.

My config looks like this in YAML and multiple routes support is working great.

```
hostname: homeassistant
auth_key: tskey-auth-xxx
advertise_routes: 10.0.0.0/24,192.168.0.0/24
userspace_networking: true
advertise_exit_node: true
accept_routes: true
```

**Comment by dalkain** [comment]
Still needed

**Comment by novirium** [comment]
PR #116 _does_ work, and I'm using an almost identical modification to be able to route both of my ethernet interfaces via Tailscale.  I'm not sure why it was closed - I'd be happy to submit another PR if that's all that's needed to get this fix in?

**Comment by lmagyar** [comment]
There is a working PR #135 that solves this issue: advertise all supported interfaces as Tailscale Subnets not only the default interface.

**Comment by DezzyMelb** [comment]
Unfortunately this is normal behaviour of the free tier of TailScale. TailScale only allows one subnet to be advertised on the free tier. If you want more than one Subnet Purchase the App/Service.

**Comment by novirium** [comment]
> Unfortunately this is normal behaviour of the free tier of TailScale. TailScale only allows one subnet to be advertised on the free tier. If you want more than one Subnet Purchase the App/Service.

This is incorrect, at least in my experience - I'm currently using a fork of this add-on that advertises both subnets from my HASS host, and both do indeed show up in the TailScale dashboard and allow me to access them remotely from another device. This is while using a free Tailscale account.

The free tier of TailScale does only allow 1 subnet router to my knowledge, but that router can handle more than 1 subnet.

**Comment by lmagyar** [comment]
Not stale, waiting for review/merge...
